.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,17 @@
+# https://pre-commit.com
+# This GitHub Action assumes that the repo contains a valid .pre-commit-config.yaml file.
+name: pre-commit
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
I think it should trigger on all branches, so when someone is doing a push to a feature branch, or we merge into the dev branch, CI/CD is always triggered to get new results?